packages/x-charts/src/BarChart/BarElement.tsx

```diff
@@ -64,6 +65,7 @@ export const BarElementPath = styled('rect', {
 export type BarElementProps = Omit<BarElementOwnerState, 'isFaded' | 'isHighlighted'> &
   React.ComponentPropsWithoutRef<'path'> & {
     highlightScope?: Partial<HighlightScope>;
+    yOrigine: number;
```

Comment:
`s/yOrigine/yOrigin/`

Reply:
I might admit I do not understand this comment  🙈

Reply:
My bad, that's the `sed` CLI command way to perform a **s**ubstitution, replacing `yOrigine` with `yOrigin`. "Origine" is French, the English word doesn't take a final e.

<img width="300" src="https://github.com/mui/mui-x/assets/1423607/519b3cdc-358f-47aa-bc3b-8122f1b430fc" />